# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/test/jsmodules/test_dynamicImportErrorMessage.html
# Commit: cc6bc73fbbfd
# Full Hash: cc6bc73fbbfd2dd796ae51f2b85c7e9c24ae2686
# Author: Alexandru Marc <amarc@mozilla.com>
# Date: 2025-09-25 05:12 +0300
# Regressor Bug: 1968890
# File Overlap Count: 14
# Overlapping Files: dom/base/test/jsmodules/chrome.toml, dom/base/test/jsmodules/test_dynamicImportErrorMessage.html, dom/script/ScriptLoadContext.cpp, dom/script/ModuleLoader.cpp, dom/base/test/jsmodules/import_404.mjs
# Description:
#   Revert "Bug 1968890 - Part 9: Add test for dynamic import. r=jonco" for causing multiple wpt failures
# ==============================================================================

diff -r e1c5be04b3a0 -r cc6bc73fbbfd dom/base/test/jsmodules/test_dynamicImportErrorMessage.html
--- a/dom/base/test/jsmodules/test_dynamicImportErrorMessage.html	Wed Sep 24 23:17:54 2025 +0000
+++ b/dom/base/test/jsmodules/test_dynamicImportErrorMessage.html	Thu Sep 25 05:12:32 2025 +0300
@@ -5,27 +5,12 @@
 <script>
   SimpleTest.waitForExplicitFinish();
 
-  let count = 0;
-  function maybeFinish() {
-    count++;
-    if (count == 2) {
-      SimpleTest.finish();
-    }
-  }
-
   // eslint-disable-next-line no-unused-vars
   async function testLoaded() {
-
     await import("./404.js").catch((error) => {
       ok(error instanceof TypeError, "Should be a TypeError.");
       ok(error.message.match(/404.js/), "Should have the filename.");
-      maybeFinish();
-    });
-
-    await import("./import_404.mjs").catch((error) => {
-      ok(error instanceof TypeError, "Should be a TypeError.");
-      ok(error.message.match(/404.js/), "Should have the filename from sub-modules");
-      maybeFinish();
+      SimpleTest.finish();
     });
   }
 </script>